### PR TITLE
created izam-mohammed.json

### PR DESCRIPTION
**Pull request by izam-mohammed**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4151"></a>

### CODE REVIEW

**Review by accodes21 — DISMISSED** [review]
Looks great

**Comment by izam-mohammed** [comment]
Please merge this !

**Comment by izam-mohammed** [comment]
@eddiejaoude please merge this, address if any mistakes

File: data/izam-mohammed.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        "group": "Socials",
+        "name": "Hashnode: For my blogs and articles",
+        "url": "https://hashnode.com/@izammohammed",
+        "icon": "SiHashnode"
+      },
+      {
+        "group": "Socials",
+        "name": "Medium: For my blogs in Medium",
+        "url": "https://medium.com/@izamdeveloper1",
+        "icon": "BsMedium"
```
Comment (kumarsonsoff3) [inline data/izam-mohammed.json:57]: `BsMedium` is not yet supported by LinkFree, you can use any one of these. So update it accordingly. 

image

**Comment by izam-mohammed** [comment]
Solved that issue @kumarsonsoff3 . Are there any other issues?

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

File: data/izam-mohammed.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
+{
+    "name": "Izam Mohammed",
+    "type": "personal",
+    "displayStatsPublic": true,
+    "bio": "Open Source Enthusiast | **Data Scientist** | Freeelancer",
+    "avatar": "https://github.com/izam-mohammed.png",
```
Comment (kumarsonsoff3) [inline data/izam-mohammed.json:6]: Our recent update no longer needs the `avatar` key. So please remove this avatar field from your json. 

Updated docs: https://linkfree.eddiehub.io/docs/quickstart
```suggestion
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

File: data/izam-mohammed.json (hunk 2 above)
Comment (izam-mohammed) [reply data/izam-mohammed.json:6]: okey

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good to merge. Thanks for your Contribution! 🎉

<p align="center"></p>

**Pull request merged**